### PR TITLE
New and improved CFP

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

**Comment by plaindocs** [comment]
Preview here: https://deploy-preview-348--dentist-canary-64055.netlify.com/conf/portland/2018/cfp/

File: docs/conf/portland/2018/cfp.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
+:template: 2018/base.html
+
+Call for proposals
+==================
+
+Hello hello, fellow documentarians! It's that time of year again: We’re very
+excited to announce that we are now accepting talk proposals for our next North
+American conference, coming up on **May 6-8, 2018, in Portland, Oregon**.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst:8]: North American > Portland?

File: docs/conf/portland/2018/cfp.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Every year, Write the Docs invites people from all across our community to come
+up on stage to share their insights and experience. Whether you’ve been writing
+documentation for decades or you’ve just started this year, we want to hear from
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: not just for writers? maybe rephrase to soemthing like "... or you're a support/engineering/role?? starting out / advocating for docs"? not sure but maybe we can discuss

File: docs/conf/portland/2018/cfp.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+**The deadline for submitting proposals is Midnight PST on January 08, 2018.**
+
+We'll let you know if your proposal has been accepted by late January.
+
+Conference goals
+----------------
+
+Write the Docs conferences are our chance to get together in-person to explore and celebrate the craft of documentation in a positive, inclusive environment.
+
+Between the talks on stage, the discussions in the unconf, and the conversations in the hallway, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: unconf > fully spell out and possibly link to unconf page?

Comment (thatdocslady) [reply docs/conf/portland/2018/cfp.rst]: also maybe add writing day instead of hallway conversations?

File: docs/conf/portland/2018/cfp.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+Our goal is to give documentarians a chance to connect and learn from each other. You'll have the chance to compare notes on what's happening in the industry, dig into questions of convention and good practice, and generally nerd out about all the things we love about documentation.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: maybe link documentarian to info page, and possibly sign-post it more for newcomers?

Comment (thatdocslady) [reply docs/conf/portland/2018/cfp.rst]: convention > conventions?
practice > practices?

File: docs/conf/portland/2018/cfp.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
+:template: 2018/base.html
+
+Call for proposals
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst:3]: general question - can we move away from mid-sentence line breaks? IIRC it doesn't make a difference in publications but it helps to have a one-sentence-per-line with appropriate soft tab enabled in text editor.

File: docs/conf/portland/2018/cfp.rst [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+Between the talks on stage, the discussions in the unconf, and the conversations in the hallway, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.
+
+Our goal is to give documentarians a chance to connect and learn from each other. You'll have the chance to compare notes on what's happening in the industry, dig into questions of convention and good practice, and generally nerd out about all the things we love about documentation.
+
+What we’re looking for
+----------------------
+
+The focus of Write the Docs is software documentation, but we actively seek
+talks that address a wide range of related subjects,
+at various levels of expertise. Documentation perspectives from other fields are welcome! Talks from the scientific community, fiction writers, system administrators, and support staff – in addition to technical writers and software programmers – are all valuable to our attendees.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: software programmers > either programmers or software developers

File: docs/conf/portland/2018/cfp.rst [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+We welcome talks from first-time speakers, from industry experts, and from
+everyone in between, and across the board, we prefer hearing about new
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: too many "ands" in the sentence, maybe break up

File: docs/conf/portland/2018/cfp.rst [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+approaches than about tried-and-tested technology. We especially welcome talks
+from underrepresented groups within the tech community.
+
+We prefer talks backed by experience and experimentation to talks about theory,
+and we definitely don't like talks that bad-mouth technologies or approaches.
+Don't tell us why you hate something – tell us how you overcame the problems it was causing.
+
+We tend to avoid talks about specific tools, which often turn into marketing
+pitches or tutorials. We would much rather hear about process, culture, data,
+people, or the metaphysical side effects of spending your life thinking about docs.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: metaphysical <3

File: docs/conf/portland/2018/cfp.rst [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+When crafting talk proposals remember that you're going to be talking to a mix
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: comma after "proposals"?

File: docs/conf/portland/2018/cfp.rst [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+up on stage to share their insights and experience. Whether you’ve been writing
+documentation for decades or you’ve just started this year, we want to hear from
+you! Read on to learn more about the goals of the conference and what we look
+for in talk proposals.
+
+In the meantime, mark your calendars:
+
+**The deadline for submitting proposals is Midnight PST on January 08, 2018.**
+
+We'll let you know if your proposal has been accepted by late January.
```
Comment (ericholscher) [inline docs/conf/portland/2018/cfp.rst]: by the end of January? Seems more final and similarly not firm :)

File: docs/conf/portland/2018/cfp.rst (hunk 3 above)
Comment (ericholscher) [inline docs/conf/portland/2018/cfp.rst]: "unconf" is maybe a bit twee. Perhaps full name and link to unconf page?

File: docs/conf/portland/2018/cfp.rst [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+
+When crafting talk proposals remember that you're going to be talking to a mix
+of levels of expertise, skill sets, and professions. Your talk doesn't have to be
+relevant to everyone, but it should be relevant to most people and shouldn't
+make too many assumptions about what people already know.
+
+Take a look at the abstracts for accepted talks from the last couple of
+conferences for some ideas:
+
+* `Portland 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst:69]: maybe sort this list as latest-first? I was confused initially

File: docs/conf/portland/2018/cfp.rst (hunk 3 above)
Comment (ericholscher) [reply docs/conf/portland/2018/cfp.rst]: Maybe don't need a link, but at least the full name, don't want to link folks off the page necessarily.

File: docs/conf/portland/2018/cfp.rst [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+* **Community mentorship** – We have an ever-growing pool of previous Write the Docs speakers, many of whom are happy to be a second pair of eyes on talk proposals. If you're interested in working with a past speaker, let us know!
+* **Meetup brainstorming** – For some in-person workshopping, check in on your `local meetup group<http://www.writethedocs.org/meetups/>`_ and see if they have a talk brainstorming session on their calendar. If they don't... ask if they're planning one!
+* **Slack hivemind** – You can also hit up the hivemind directly on the Write the Docs Slack, any time of day! (If you're not registered yet, you can at `http://slack.writethedocs.org/<http://slack.writethedocs.org/>`_.)
+
+Presentation format
+-------------------
+
+Presentations will be scheduled in 30-minute blocks. We won't be doing Q&A after
+the talks, but we ask that speakers be available at the front of the stage
+for a few minutes of the break after their talk to answer questions.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst:88]: maybe add that you can pitch an unconference session about your talk at the end?

File: docs/conf/portland/2018/cfp.rst [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+
+Speaker benefits & logistics
+----------------------------
+
+If you are selected to speak at Write the Docs, we will waive your attendance
+fee. We regret that we are not able to cover the cost of travel or lodging for
+all speakers at this time. By not covering travel costs for our speakers, we are
+able to keep ticket prices low and to make the conference accessible to the
+largest number of participants possible. If your proposal is accepted but you
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: 1 - move "If your proposal" to a new para
2 - maybe add "However" or something to indicate that this is not a hard requirement

File: docs/conf/portland/2018/cfp.rst [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+are unable to attend due to travel costs, please let us know and we will do our
+best to help make it happen.
+
+Finally, you’ll hear from us with our proposal decisions by late January.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: no need for "Finally"?

File: docs/conf/portland/2018/cfp.rst [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+
+Questions?
+----------
+
+If you have any questions, please email us at `conf@writethedocs.org <mailto:conf@writethedocs.org>`_ and let us know.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: change conf@ to portland@

File: docs/conf/portland/2018/cfp.rst [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+* `Portland 2016 <http://www.writethedocs.org/conf/na/2016/speakers/>`_
+* `Prague 2017 <http://www.writethedocs.org/conf/eu/2017/speakers/>`_
+* `Prague 2016 <http://www.writethedocs.org/conf/eu/2016/speakers/>`_
+
+Need help?
+-----------
+
+If you need a hand preparing your talk proposal, there are lots of good places to start:
+
+* **Community mentorship** – We have an ever-growing pool of previous Write the Docs speakers, many of whom are happy to be a second pair of eyes on talk proposals. If you're interested in working with a past speaker, let us know!
```
Comment (ericholscher) [inline docs/conf/portland/2018/cfp.rst:79]: We should probably direct them to slack instead of us, unless we have a specific plan here.

File: docs/conf/portland/2018/cfp.rst [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+best to help make it happen.
+
+Finally, you’ll hear from us with our proposal decisions by late January.
+
+Questions?
+----------
+
+If you have any questions, please email us at `conf@writethedocs.org <mailto:conf@writethedocs.org>`_ and let us know.
+
+Form
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: change to "Submit your proposal"? more clear

File: docs/conf/portland/2018/cfp.rst [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+----
+
+You can also view `our CFP <https://docs.google.com/forms/d/e/1FAIpQLSdb-MNRT3TdDTuVIcf-3lZRWDOgguZfjatLr-HLN5ngx5t04w/viewform>`_ in its own page.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst]: move this under the embedded form?

**Review by thatdocslady — COMMENTED** [review]
Looks good! Some nit-picky comments/suggestions :)

File: docs/conf/portland/2018/cfp.rst (hunk 16 above)
Comment (ericholscher) [reply docs/conf/portland/2018/cfp.rst:79]: Like, we should ask for mentors in advance, and publish them (good example here: https://www.pycascades.com/speakers/)

File: docs/conf/portland/2018/cfp.rst [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+Speaker benefits & logistics
+----------------------------
+
+If you are selected to speak at Write the Docs, we will waive your attendance
+fee. We regret that we are not able to cover the cost of travel or lodging for
+all speakers at this time. By not covering travel costs for our speakers, we are
+able to keep ticket prices low and to make the conference accessible to the
+largest number of participants possible. If your proposal is accepted but you
+are unable to attend due to travel costs, please let us know and we will do our
+best to help make it happen.
```
Comment (ericholscher) [inline docs/conf/portland/2018/cfp.rst]: I think this can maybe be worded differently. I think more of a "We don't cover costs for all our our speakers to keep prices lower, but have a travel assistance fund for people who need it". I feel like we bury the lede a bit.

File: docs/conf/portland/2018/cfp.rst (hunk 15 above)
Comment (ericholscher) [inline docs/conf/portland/2018/cfp.rst]: Should make the `portland@`

**Review by ericholscher — COMMENTED** [review]
Looks great overall. I just had a few nits. I think it could also use some bolding or highlighting, to make it a bit easier to skim and pull out the relevant bits.

File: docs/conf/portland/2018/cfp.rst (hunk 5 above)
Comment (ericholscher) [reply docs/conf/portland/2018/cfp.rst:3]: +1 makes it a lot easier to review, though I still like line-break on punctuation over sentence, because often commas seperate ideas :) It falls down on lists though, so _shrug_.

Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst:3]: I have a preference for short lines, esp on a small screen, but can go with the majority.

File: docs/conf/portland/2018/cfp.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst:8]: As a description seems fine, esp with Portland in the bold text.

File: docs/conf/portland/2018/cfp.rst (hunk 2 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: I'm happy to leave as is, all documentarians write docs innit? Just not all the time.

File: docs/conf/portland/2018/cfp.rst (hunk 3 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: Fixing, No links as there is nothing to link to yet.

File: docs/conf/portland/2018/cfp.rst (hunk 6 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: Fixing

File: docs/conf/portland/2018/cfp.rst (hunk 7 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: :+1:

File: docs/conf/portland/2018/cfp.rst (hunk 9 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: :+1:

File: docs/conf/portland/2018/cfp.rst (hunk 12 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst:88]: We can mention that in the acceptance email / form

File: docs/conf/portland/2018/cfp.rst (hunk 13 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: confused about 2

File: docs/conf/portland/2018/cfp.rst (hunk 17 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: :+1:

File: docs/conf/portland/2018/cfp.rst (hunk 15 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: :+1:

File: docs/conf/portland/2018/cfp.rst (hunk 18 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst]: :+1:

File: docs/conf/portland/2018/cfp.rst (hunk 11 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst:69]: Fixed

File: docs/conf/portland/2018/cfp.rst (hunk 5 above)
Comment (plaindocs) [reply docs/conf/portland/2018/cfp.rst:3]: I also prefer short lines for diffs, oddly enough.

File: docs/conf/portland/2018/cfp.rst (hunk 2 above)
Comment (kelly-o) [reply docs/conf/portland/2018/cfp.rst]: I changed to 'working on', if that's cool.

File: docs/conf/portland/2018/cfp.rst (hunk 4 above)
Comment (kelly-o) [reply docs/conf/portland/2018/cfp.rst]: if it's 'question of' they need to be singular, but we can recast if it bugs you

File: docs/conf/portland/2018/cfp.rst [hunk 20]
```diff
@@ -0,0 +1,116 @@
+:template: 2018/base.html
+
+Call for proposals
```
Comment (thatdocslady) [inline docs/conf/portland/2018/cfp.rst:3]: Ah forgot to ask this, do we want to title-cap or sentence cap? usually H1's get title-caps

**Review by thatdocslady — COMMENTED** [review]
Amazeballs! Thanks @plaindocs @kelly-o for this. One last nit-pick but I love the new breakdowns :)

File: docs/conf/portland/2018/cfp.rst (hunk 20 above)
Comment (kelly-o) [reply docs/conf/portland/2018/cfp.rst:3]: I'm not picky, so long as we're consistent (sentence case is my default, atm, but if you'd rather title, that's fine by me)

**Comment by ericholscher** [comment]
This ready to be merged? 🎆 ?

**Comment by plaindocs** [comment]
Yup
